packages/core/src/validate.ts

```diff
@@ -447,3 +447,15 @@ function* getEnumErrors(contractDef: ContractDefinition, decodeSrc: SrcDecoder):
     };
   }
 }
+
+let silenced = false;
+
+export function silenceWarnings(): void {
```

Comment:
I think that this should forbid subsequent calls to prevent multiple logs, in line with the feature.

Reply:
I changed this so that subsequent calls will not print the warning. I thought forbidding it with an error was too strong, and some people may want to use the function in different files.